ui/app/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Removed the separate license in 9c1fc6c. @stuartnelson3 could you please approve that this happened with your agreement?